version/1/4/fr/code_of_conduct.md

```diff
@@ -25,18 +25,18 @@ Exemples de comportements non acceptables :
 * l'utilisation de langage ou d'imagerie sexualisés et les avances sexuelles
 non sollicitées
 * le _trolling_, les commentaires insultants ou désobligeants, et les
-attaques personnelles ou d'ordre politique
+attaques personnelles ou politiques
```

Comment:
I think "political attacks" is open to debate[1], but "of political matters" is broader.
Everything is political (including skin color or sexuality) so I think we should let that line as it currently is.

[1] some can argue that political would only be about politicians, laws and parties